opm/simulators/wells/StandardWellPrimaryVariables.cpp

```diff
@@ -741,5 +741,7 @@ INSTANCE(BlackOilIndices<0u,0u,0u,0u,true,false,0u,0u>)
 INSTANCE(BlackOilIndices<0u,0u,0u,0u,false,true,0u,0u>)
 INSTANCE(BlackOilIndices<0u,0u,0u,1u,false,true,0u,0u>)
 INSTANCE(BlackOilIndices<0u,0u,0u,1u,false,false,1u,0u>)
+INSTANCE(BlackOilIndices<0u,1u,0u,0u,true,false,0u,0u>)
+INSTANCE(BlackOilIndices<1u,0u,0u,0u,true,false,0u,0u>)
```

Comment:
@akva2 , can you have a look at the part related to `BlackOilIndices`? thanks.

Reply:
only thing I find surprising is that we need two new instances. It's just one new simulator?

Reply:
I used the code to also test the foam + the extended blackoil module (an improved solvent option), but dropped it. I will remove the redundant lines.